src/deserialization.hpp

```diff
@@ -133,11 +133,11 @@ Status deserializePredictRequest(
         // OV can throw exceptions derived from std::logic_error.
     } catch (const InferenceEngine::details::InferenceEngineException& e) {
         Status status = StatusCode::OV_INTERNAL_DESERIALIZATION_ERROR;
-        SPDLOG_ERROR("{}: {}", status.string(), e.what());
+        SPDLOG_DEBUG("{}: {}", status.string(), e.what());
         return status;
     } catch (std::logic_error& e) {
         Status status = StatusCode::OV_INTERNAL_DESERIALIZATION_ERROR;
-        SPDLOG_ERROR("{}: {}", status.string(), e.what());
+        SPDLOG_DEBUG("{}: {}", status.string(), e.what());
```

Comment:
in this case it makes sense to use spdlog::debug (we loose the line & file but we still can debug OVMS with different CLI option without need to rebuild (request that from client).